### PR TITLE
Topic/metagenomics improvements

### PR DESCRIPTION
**Pull request by bernt-matthias**
added links and some reformulations in the general metagenomics tutorial. See also https://github.com/galaxyproject/training-material/issues/530

There are a few TODOs left in the text for which I could need input.

### CODE REVIEW

File: topics/metagenomics/tutorials/general-tutorial/tutorial.md [hunk 1]
```diff
@@ -385,7 +385,7 @@ Have a look at the taxonomy output. You will see that every read now has a class
 
 The next step is then to use this information to know the abundance of the different found taxons.
 
-> ### {% icon hands_on %} Hands-on: Cluster our data into OTUs
+> ### {% icon hands_on %} Hands-on: Cluster our data into OTUs TODO how is the taxonomy used here? Are actually only the phylotypes clustered into OTUs?
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/general-tutorial/tutorial.md]: did you find the answer to this TODO? (could you put it as a comment (`<!-- -->`) if not?)

Comment (shiltemann) [reply topics/metagenomics/tutorials/general-tutorial/tutorial.md]: (sorry, didnt see your comment about the TODOs in description) I will see if I can find the answer to this

**Comment by bgruening** [comment]
@bernt-matthias nice contribution! Its great to see your contributions everywhere. Keep up the wonderful work!

Can you resolve the conflicts here, please.

**Comment by bernt-matthias** [comment]
Resolved it, but some TODOs remain. @shiltemann could you find out what is going on in the "Cluster our data into OTUs" step?

**Comment by bernt-matthias** [comment]
@shiltemann any progress on my questions?

**Comment by shiltemann** [comment]
@bernt-matthias I think how the clustering works is as follows:

- all individual sequences are classified, and get assigned a confidence score (0-100%)
- next, sequences are grouped at 97% identity threshold (not using taxonomy info)
- then, for each cluster, a consensus classification is determined based on the classification of the individual sequences and taking their confidence scores into account

**Comment by bernt-matthias** [comment]
Thanks @shiltemann . I added your explanations. For now I will not have time to continue on this PR. I will give the tutorial again in December -- I hope that I can continue to improve the tutorial.

**Comment by bernt-matthias** [comment]
In the December course I decided to use the "16S Microbial Analysis with Mothur" tutorial. So I do not have any further changes at the moment.

**Pull request merged**